### PR TITLE
Update release notes for WAFS removal from GFS

### PR DESCRIPTION
**Pull request by KateFriedman-NOAA**
# Description

This PR updates the release notes for the WAFS removal from the GFS. Files and settings that are removed are noted.

Also update `gfs_ver=v16.3.19`.

@YaliMao-NOAA please let me know if the release notes are complete, especially the list of output files being removed, thanks!

Refs #2591

# Type of change

Production update

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO
- Does this change require an update to any of the following submodules? YES - WAFS is removed

### CODE REVIEW

**Review by aerorahul — APPROVED** [review]
looks good.

File: docs/Release_Notes.md
Comment (YaliMao-NOAA) [inline docs/Release_Notes.md:66]: Should the jobs be listed here?

File: docs/Release_Notes.md
Comment (YaliMao-NOAA) [inline docs/Release_Notes.md]: Some files are missing. Please refer column C in this document for a complete list: https://docs.google.com/spreadsheets/d/1Nt343Z9x9UycweFik3HRFpXkqIjs7m20s15yGOhsgUY

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Comment by KateFriedman-NOAA** [comment]
@YaliMao-NOAA reminder to review this PR, thanks!

File: docs/Release_Notes.md
Comment (YaliMao-NOAA) [inline docs/Release_Notes.md:1]: @KateFriedman-NOAA I left some comments on Monday. May you please take a look?

**Comment by YaliMao-NOAA** [comment]
@KateFriedman-NOAA As in the recent [WAFS PR#59](https://github.com/NOAA-EMC/WAFS/pull/59) , WAFS master file (old name: gfs.tCCz.wafs.grb2fFFF, new name: wafs.tCCz.master.fFFF.grib2) at forecast hour F001-F005 will be removed too.

**Comment by KateFriedman-NOAA** [comment]
@YaliMao-NOAA Thanks for the information. I already had that file listed in the release notes but have added text to state it is being renamed in the WAFS package. Please review and let me know if the release notes are good to go. Once I have your approval I will merge this and cut a hand-off tag. Thanks!

**Review by YaliMao-NOAA — APPROVED** [review]

File: docs/Release_Notes.md
Comment (KateFriedman-NOAA) [reply docs/Release_Notes.md:1]: @YaliMao-NOAA Thanks for the feedback! I have updated the release notes to add a list of the jobs removed and to add the missing files from the file list. Please review and let me know if anything else needs updating. Thanks!

**Comment by YaliMao-NOAA** [comment]
@KateFriedman-NOAA 
1. wmo/xtrn.wfsgfs0006.gfs_atmos_wafs_f30_00 is still in production.
2. Please add this line under the products to be retired:
wafs.tCCz.master.fFFF.grib2 where FFF is from 001 to 005

**Comment by KateFriedman-NOAA** [comment]
> 1. wmo/xtrn.wfsgfs0006.gfs_atmos_wafs_f30_00 is still in production.
> 2. Please add this line under the products to be retired:
wafs.tCCz.master.fFFF.grib2 where FFF is from 001 to 005

@YaliMao-NOAA updated @ 98b902b8

**Review by YaliMao-NOAA — APPROVED** [review]
Looks good.

**Comment by KateFriedman-NOAA** [comment]
Thanks @YaliMao-NOAA ! I will now merge this into the auth repo release branch.

**Pull request merged**
